ceremonies/src/tests.rs

```diff
@@ -415,7 +421,7 @@ fn attest_attendees_for_non_participant_fails_silently() {
 }
 
 #[test]
-fn attest_attendee_for_non_participant_fails() {
```

Comment:
this test didn't test what it claimed to test.

I now made two out of one, to trigger both possible errors